apps/nestjs-wallet/docs/tutorial.md

```diff
@@ -1,5 +1,56 @@
 # Tutorial
 
+## Business overview
+The business objective of this tutorial is to demonstrate how to issue a proof in the form of a Verifiable Credential (VC) to a known user connected to a portal that we will call "authority portal".
+In the example below, we will issue a permanent residency card to a user we will call "citizen". We use the context "https://w3id.org/citizenship/v1" which describes the data of a VC of type "PermanentResidentCard".
+
+Here is an extract of the context:
+
+        ...
+        "id": "@id",
+        "type": "@type",
+
+        "ctzn": "https://w3id.org/citizenship#",
+        "schema": "http://schema.org/",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+        "birthCountry": "ctzn:birthCountry",
+        "birthDate": {"@id": "schema:birthDate", "@type": "xsd:dateTime"},
+        "commuterClassification": "ctzn:commuterClassification",
+        "familyName": "schema:familyName",
+        "gender": "schema:gender",
+        "givenName": "schema:givenName",
+        "lprCategory": "ctzn:lprCategory",
+        "lprNumber": "ctzn:lprNumber",
+        "residentSince": {"@id": "ctzn:residentSince", "@type": "xsd:dateTime"}
+        ...
+
+To get the definition of the field "gender" which points to the definition "schema:gender", use the URI of "schema" which points to "http://schema.org/", so we will have a definition available at "http://schema.org/gender".
+TODO interpretation of id and type.
+
+We therefore assume that the citizen is connected to the autority portal and that the information contained in the context about the citizen is available in the autority portal database.
+
+Business workflow:
+- The citizen requests a "PermanentResidentCard" proof on the autority portal
+- The autority portal displays a QR code for the citizen to scan with his mobile wallet.
+- The citizen's mobile wallet contacts the autority portal to prove that he has a DID
+- The citizen's mobile wallet contacts the autority portal again to receive the "PermanentResidentCard"
+- The autority portal responds by sending the "PermanentResidentCard" proof containing the citizen's information signed by the autority portal.
+
+## Technical overview
+From a technical point of view, in this tutorial, we have access to the server API but no mobile wallet is available. So we will use the server API for the portal and the mobile wallet.
```

Comment:
I'm a bit confused by this line. It says that there is "no mobile wallet available" but then says that "we will use the server API for the portal and the mobile wallet". How can we use the mobile wallet if there is none available?

Reply:
I mean ; we use the server API also in place of the mobile wallet

Reply:
Ah, I see now. What do you think of this suggestion?
```suggestion
From a technical point of view, in this tutorial, we have access to the server API but no mobile wallet is available. So we will use the server API for both roles of the portal and the mobile wallet.
```